### YamlMime:ManagedReference
items:
- uid: System.Data.Linq.DuplicateKeyException
  id: DuplicateKeyException
  children:
  - System.Data.Linq.DuplicateKeyException.#ctor(System.Object)
  - System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String)
  - System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String,System.Exception)
  - System.Data.Linq.DuplicateKeyException.Object
  langs:
  - csharp
  name: DuplicateKeyException
  nameWithType: DuplicateKeyException
  fullName: System.Data.Linq.DuplicateKeyException
  type: Class
  summary: "Utlöses när ett försök görs att lägga till ett objekt i cacheminnet identitet med hjälp av en nyckel som redan används."
  syntax:
    content: 'public class DuplicateKeyException : InvalidOperationException'
  inheritance:
  - System.InvalidOperationException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object)
  id: '#ctor(System.Object)'
  parent: System.Data.Linq.DuplicateKeyException
  langs:
  - csharp
  name: DuplicateKeyException(Object)
  nameWithType: DuplicateKeyException.DuplicateKeyException(Object)
  fullName: System.Data.Linq.DuplicateKeyException.DuplicateKeyException(Object)
  type: Constructor
  assemblies:
  - System.Data.Linq
  namespace: System.Data.Linq
  summary: "Initierar en ny instans av den <xref href=&quot;System.Data.Linq.DuplicateKeyException&quot;> </xref> klass."
  syntax:
    content: public DuplicateKeyException (object duplicate);
    parameters:
    - id: duplicate
      type: System.Object
      description: "Dubblettnyckel som orsakade det undantagsfel."
  overload: System.Data.Linq.DuplicateKeyException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String)
  id: '#ctor(System.Object,System.String)'
  parent: System.Data.Linq.DuplicateKeyException
  langs:
  - csharp
  name: DuplicateKeyException(Object,String)
  nameWithType: DuplicateKeyException.DuplicateKeyException(Object,String)
  fullName: System.Data.Linq.DuplicateKeyException.DuplicateKeyException(Object,String)
  type: Constructor
  assemblies:
  - System.Data.Linq
  namespace: System.Data.Linq
  summary: "Initierar en ny instans av den <xref href=&quot;System.Data.Linq.DuplicateKeyException&quot;> </xref> klassen genom att referera till dubblettnyckel och ett felmeddelande."
  syntax:
    content: public DuplicateKeyException (object duplicate, string message);
    parameters:
    - id: duplicate
      type: System.Object
      description: "Dubblettnyckel som orsakade det undantagsfel."
    - id: message
      type: System.String
      description: "Meddelandet visas när den undantag."
  overload: System.Data.Linq.DuplicateKeyException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String,System.Exception)
  id: '#ctor(System.Object,System.String,System.Exception)'
  parent: System.Data.Linq.DuplicateKeyException
  langs:
  - csharp
  name: DuplicateKeyException(Object,String,Exception)
  nameWithType: DuplicateKeyException.DuplicateKeyException(Object,String,Exception)
  fullName: System.Data.Linq.DuplicateKeyException.DuplicateKeyException(Object,String,Exception)
  type: Constructor
  assemblies:
  - System.Data.Linq
  namespace: System.Data.Linq
  summary: "Initierar en ny instans av den <xref href=&quot;System.Data.Linq.DuplicateKeyException&quot;> </xref> klassen genom att referera till dubblettnyckel, tillhandahåller ett felmeddelande och ange undantag som orsakade det här undantagsfel."
  syntax:
    content: public DuplicateKeyException (object duplicate, string message, Exception innerException);
    parameters:
    - id: duplicate
      type: System.Object
      description: "Dubblettnyckel som orsakade det undantagsfel."
    - id: message
      type: System.String
      description: "Meddelandet visas när den undantag."
    - id: innerException
      type: System.Exception
      description: "Tidigare undantaget som orsakat den <xref href=&quot;System.Data.Linq.DuplicateKeyException&quot;> </xref> undantagsfel."
  overload: System.Data.Linq.DuplicateKeyException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Linq.DuplicateKeyException.Object
  id: Object
  parent: System.Data.Linq.DuplicateKeyException
  langs:
  - csharp
  name: Object
  nameWithType: DuplicateKeyException.Object
  fullName: System.Data.Linq.DuplicateKeyException.Object
  type: Property
  assemblies:
  - System.Data.Linq
  namespace: System.Data.Linq
  summary: "Hämtar det objekt som orsakade undantaget."
  syntax:
    content: public object Object { get; }
    return:
      type: System.Object
      description: "Det objekt som orsakade undantaget."
  overload: System.Data.Linq.DuplicateKeyException.Object*
  exceptions: []
  platform:
  - net462
references:
- uid: System.InvalidOperationException
  isExternal: false
  name: System.InvalidOperationException
- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object)
  parent: System.Data.Linq.DuplicateKeyException
  isExternal: false
  name: DuplicateKeyException(Object)
  nameWithType: DuplicateKeyException.DuplicateKeyException(Object)
  fullName: System.Data.Linq.DuplicateKeyException.DuplicateKeyException(Object)
- uid: System.Object
  parent: System
  isExternal: true
  name: Object
  nameWithType: Object
  fullName: System.Object
- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String)
  parent: System.Data.Linq.DuplicateKeyException
  isExternal: false
  name: DuplicateKeyException(Object,String)
  nameWithType: DuplicateKeyException.DuplicateKeyException(Object,String)
  fullName: System.Data.Linq.DuplicateKeyException.DuplicateKeyException(Object,String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Data.Linq.DuplicateKeyException.#ctor(System.Object,System.String,System.Exception)
  parent: System.Data.Linq.DuplicateKeyException
  isExternal: false
  name: DuplicateKeyException(Object,String,Exception)
  nameWithType: DuplicateKeyException.DuplicateKeyException(Object,String,Exception)
  fullName: System.Data.Linq.DuplicateKeyException.DuplicateKeyException(Object,String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Data.Linq.DuplicateKeyException.Object
  parent: System.Data.Linq.DuplicateKeyException
  isExternal: false
  name: Object
  nameWithType: DuplicateKeyException.Object
  fullName: System.Data.Linq.DuplicateKeyException.Object
- uid: System.Data.Linq.DuplicateKeyException.#ctor*
  parent: System.Data.Linq.DuplicateKeyException
  isExternal: false
  name: DuplicateKeyException
  nameWithType: DuplicateKeyException.DuplicateKeyException
- uid: System.Data.Linq.DuplicateKeyException.Object*
  parent: System.Data.Linq.DuplicateKeyException
  isExternal: false
  name: Object
  nameWithType: DuplicateKeyException.Object
